src/staking-v3/hooks/useAprV3.ts

```diff
@@ -24,9 +22,8 @@ export const useAprV3 = () => {
     return Number(ethers.utils.formatEther(String(wei)));
   };
 
-  // Todo: fetch it via API
   const periodsPerCycle = computed<number>(() => {
```

Comment:
You can simplify the expression
```
const periodsPerCycle = computed<number>(() => eraLengths.value.periodsPerCycle);
```

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1167/commits/6ed5bcc593e571d4730b48be35f47dc8f0a2ac29) commit